tests/system/aiplatform/test_resources/cpr_user_code/predictor.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import joblib
+import numpy as np
+from typing import Any
+
+from google.cloud import storage
+from google.cloud.aiplatform.prediction.predictor import Predictor
+
+
+class SklearnPredictor(Predictor):
+    """Interface for Predictor class that users would be implementing."""
+
+    def __init__(self):
```

Comment:
is it necessary to override `__init__` if it has no effect?

Reply:
`__init__` in predictor template is marked as `NotImplementedError` by default.